umap/static/umap/js/umap.xhr.js

```diff
@@ -95,7 +95,7 @@ L.U.Xhr = L.Evented.extend({
             // 0 === request cut by user or CORS
             self.ui.alert({
               content: L._(
-                'Issue reaching that URL (network problem or CORS protection)'
+                'Issue reaching that URL (network problem or CORS protection)' + ` ${settings.uri}`
```

Comment:
```suggestion
                `Issue reaching that URL (network problem or CORS protection) ${settings.uri}`
```

Reply:
Why not, even if this means that every languages would need an update, but in that case I'd better add a variable, to keep the string "simple" for translators. I'll do that :)